### PR TITLE
docs(storybook): removes storybook docs from dependabot updates

### PR DESCRIPTION
**Pull request by thatguynamedandy**
Temporarily removing @storybook scope modules from the dependabot updates to reduce the amount of noise. Once we
have migrated to storybook 6 we should revert this change..

re #35

- [x] The commit messages follow the [convention for this project](./blob/master/docs/CONTRIBUTING.md#conventional-commits)

### CODE REVIEW

**Review by elenagarrone — DISMISSED** [review]

**Review by owensgit — APPROVED** [review]
Good idea Andy 👍

File: .github/dependabot.yml [hunk 1]
```diff
@@ -23,3 +23,6 @@ updates:
       prefix: 'npm'
       include: 'scope'
     versioning-strategy: increase
+    ignored_updates:
```
Comment (elenagarrone) [inline .github/dependabot.yml:26]: Are you sure that's right?
In our project we use:

```
ignore:
      - dependency-name: ''
```

**Comment by thatguynamedandy** [comment]
Just as I did this I noticed a storybook 5.3.x upgrade which I assumed wasn't going to happen. I might see if I can rewrite it to only ignore 6.x.x version

File: .github/dependabot.yml (hunk 1 above)
Comment (elenagarrone) [reply .github/dependabot.yml:26]: Here are the GitHub docs: https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#ignore

**Review by elenagarrone — COMMENTED** [review]
Please see my comment

File: .github/dependabot.yml (hunk 1 above)
Comment (thatguynamedandy) [reply .github/dependabot.yml:26]: good question, I think I followed the docs correctly..
https://dependabot.com/docs/config-file/#ignored_updates

Comment (elenagarrone) [reply .github/dependabot.yml:26]: I'm pretty sure the Dependabot config (and docs) are the same as the Dependabot config/docs for GitHub. 
As far as I know the Github dependabot as less configs.

**Review by elenagarrone — APPROVED** [review]
👍

**Review by owensgit — APPROVED** [review]

**Pull request merged**